### PR TITLE
[JENKINS-60866] Un-inline JavaScript/Style in Job views

### PR DESCRIPTION
**Pull request by Wadeck**
See [JENKINS-60866](https://issues.jenkins-ci.org/browse/JENKINS-60866).

As `${imagesURL}` is often used, putting it in the head tag directly will ease lots of other un-inline PRs. 
Similar to previous such PRs, I take the opportunity to adjust / make consistent the jelly namespace and similar inline style code.

To access the "buildTimeTrend" view, you have to run a build, then enter that run page. On the right you will see "Took x seconds", click on that duration. (First time I saw that page 🤣)

<details>

<summary>Screenshot for the style change</summary>

#### Before

Screenshot-2020-04-05_17-44-02

#### After, (useless?) padding-top removed on the table

Screenshot-2020-04-05_17-44-07

---

</details>

### Proposed changelog entries

* (internal) Remove inline resources from Job views

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a JIRA issue should exist and be labeled as `lts-candidate`

### CODE REVIEW

File: core/src/main/resources/hudson/model/Job/buildTimeTrend.jelly [hunk 1]
```diff
@@ -25,49 +25,27 @@ THE SOFTWARE.
...
           <div align="center">
-            <img src="buildTimeGraph/png" width="500" height="400" lazymap="buildTimeGraph/map" alt="[${%Build time graph}]" style="float:right"/>
+            <img class="build-time-graph" src="buildTimeGraph/png" width="500" height="400" lazymap="buildTimeGraph/map" alt="[${%Build time graph}]" />
           </div>
 
           <j:set var="isMasterSlaveEnabled" value="${!empty(app.slaves)}"/>
           <div align="center">
-            <script>
-                function displayBuilds(data) {
-                    var p = $$('trend');
```
Comment (Wadeck) [inline core/src/main/resources/hudson/model/Job/buildTimeTrend.jelly:45]: FTR $$ => $ inside Jelly :)

Comment (fqueiruga) [reply core/src/main/resources/hudson/model/Job/buildTimeTrend.jelly:45]: Oh wow, nice of you to remove that one

**Review by fqueiruga — APPROVED** [review]
Nice PR!

Great that you're also trying to migrate off prototype. The Element class is also from prototype but I think it would be too risky to swap it with HTMLElement, too error prone for the scope of this PR.

**Comment by Wadeck** [comment]
> The Element class is also from prototype

Yeah the things that are potentially breaking, I prefer to keep them "as is" ;)

**Comment by fqueiruga** [comment]
Yeah, that's too sensitive for this PR IMO

**Review by res0nance — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by res0nance** [comment]
It seems that the conversations here have all been resolved. Marking this ready-for-merge. Will merge in 24h if no negative feedback

**Pull request merged**